CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 # vim: et ts=4 sts=4 sw=4 tw=0
 
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
```

Comment:
Can this line move down a bit? I think there should be a better place.

Reply:
I moved it down. If you have somewhere specific you want it let me know.